modules/getting-started/ref-rhdh-sizing.adoc

```diff
@@ -1,28 +1,33 @@
 [id='ref-rhdh-sizing_{context}']
 = Sizing requirements for {product}
 
-Scalability of {product} requires significant resource allocation. The following table lists the sizing requirements for installing and running {product}, including both the {product-short} application and {product-short} database components.
+Scalability of {product} requires significant resource allocation. The following table lists the sizing requirements for installing and running {product}, including {product-short} application, database components, and Operator.
 
 .Recommended sizing for running {product}
-[cols="40%,30%,30%", frame="all", options="header"]
+[cols="25%,25%,25%,25%", frame="all", options="header"]
 |===
 |Components
 |{product} application
 |{product} database
+|{product} Operator
 
 |Central Processing Unit (CPU)
 |4 vCPU
 |2 vCPU
+|1 vCPU
 
 |Memory
 |16 GB
 |8 GB
+|256 Mi
```

Comment:
```suggestion
|1500 Mi
```

Based on what we have in https://github.com/redhat-developer/rhdh-operator/blob/main/.rhdh/bundle/manifests/rhdh-operator.clusterserviceversion.yaml#L283, the memory limit of the controller container has been increased to 1 Gi. Taking into account the kube-rbac-proxy container limit set to [128 Mi](https://github.com/redhat-developer/rhdh-operator/blob/main/.rhdh/bundle/manifests/rhdh-operator.clusterserviceversion.yaml#L239), I think we should update this.